docs/api_walkthrough.rst

```diff
@@ -1,34 +1,29 @@
 Getting started with the Raiden API
 ###################################
 
-
-Contents:
-
 .. toctree::
-   :maxdepth: 2
-
-
+  :maxdepth: 3
 
 Introduction
-*************
-Raiden has a Restful API with URL endpoints corresponding to actions that the user can perform with his channels. The endpoints accept and return JSON encoded objects. The api url path always contains the api version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version.
+=============
+Raiden has a Restful API with URL endpoints corresponding to actions that the user can perform with its channels. The endpoints accept and return JSON encoded objects. The API URL path always contains the API version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version.
 
 In this guide we will walk through the steps necessary in order to participate in a Raiden Token Network. We will provide some different scenarios such as joining an already existing token network, registering a new token network, together with opening, closing and settling channels.
 
-Before you get started with below guides, please see :doc:`Overview and Guide <Overview-And-Guide>`, to make sure that you are connected to Raiden.
+Before you get started with below guides, please see :doc:`Overview and Guide <overview_and_guide>`, to make sure that you are connected to Raiden.
 
-Furthermore, to see all available endpoints, please see :doc:`REST API Endpoints <Rest-Api>`.
+Furthermore, to see all available endpoints, please see :doc:`REST API Endpoints <rest_api>`.
 
 
 Scenarios
-*********
-Below you'll find a series of different scenarios showing different ways that the Raiden API can be used and interacted with.
+=========
+Below is a series of different scenarios showing different ways that the Raiden API can be used and interacted with.
 
-A good way to check that you started Raiden correctly before proceeding is to check that your Raiden address is the same address as the Ethereum address that you chose, when starting the Raiden node::
+A good way to check that we started Raiden correctly before proceeding is to check that our Raiden address is the same address as the Ethereum address that we chose, when starting the Raiden node::
```

Comment:
we is quite personal. You is more neutral. It can be either you singular or plural. Even more neutral is passive voice. But I still think leaving it as `you` is fine. Changing it to `we` is not.